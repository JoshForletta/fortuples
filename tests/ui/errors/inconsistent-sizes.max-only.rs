use fortuples::fortuples;

struct Container<T>(T);

fortuples! {
    #[tuples::max_size(0)]
    impl Container<#Tuple> {}
}

fn main() {}
