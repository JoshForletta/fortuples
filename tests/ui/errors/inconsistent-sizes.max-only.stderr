error: `min_size` should be strictly less than `max_size`. Note: the default min size is 0
 --> tests/ui/errors/inconsistent-sizes.max-only.rs:5:1
  |
5 | / fortuples! {
6 | |     #[tuples::max_size(0)]
7 | |     impl Container<#Tuple> {}
8 | | }
  | |_^
  |
  = note: this error originates in the macro `fortuples` (in Nightly builds, run with -Z macro-backtrace for more info)
